public/content/lessons/2017/hardest-problem/index.mdx

```diff
@@ -51,73 +51,73 @@ Suppose you choose three random points on a circle.
 It's always helpful to name things, so let's call these fellows $P_1$, $P_2$, and $P_3$.
 What's the probability that the triangle formed by these points contains the center of the circle?
 
-{{< figure image="2D2.png" />}}
+<Figure image="2D2.png" />
 
 This is still a hard question, so you should ask yourself if there's a way to simplify the question even further.
 We still need a foothold, something to build up from.
 Maybe you imagine fixing $P_1$ and $P_2$ in place, only letting $P_3$ vary.
 In doing this, you might notice that there's special region, a certain arc, where when $P_3$ is in that arc, the triangle contains the circle's center.
 
-{{< figure video="2d_2.mp4" />}}
+<Figure video="2d_2.mp4" />
 
 Specifically, if you draw a lines from $P_1$ and $P_2$ through the center, these lines divide the circle into 4 different arcs.
 If $P_3$ happens to be in the arc opposite $P_1$ and $P_2$, the triangle will contain the center.
 Otherwise, you're out of luck.
 
-{{< figure image="arcs.png" />}}
+<Figure image="arcs.png" />
 
 It's certainly easier to visualize now, but can you answer the question?
 
-{{< question
-  question="If $P_1$ and $P_2$ are fixed in place on the circle, with an arc length $\alpha$ between them, and $P_3$ is a point chosen randomly with uniform on the circle, what is the probability that the triangle $\Delta P_1 P_2 P_3$ contains the center of the circle?"
+<Question
+  question="If $P_1$ and $P_2$ are fixed in place on the circle, with an arc length $\alpha$ between them, and $P_3$ is a point chosen randomly on the circle (with a uniform distribution), what is the probability that the triangle $\Delta P_1 P_2 P_3$ contains the center of the circle?"
   choice1="$\alpha$"
   choice2="$\alpha / 360^{\circ}$"
   choice3="$180^{\circ} - \alpha$"
   choice4="$(180^{\circ} - \alpha) / 360^{\circ}$"
-  correct=2
+  answer={2}
   explanation="As mentioned above, the point $P_3$ needs to land in the arc opposite $P_1$ and $P_2$. This arc has the same length as the one between $P_1$ and $P_2$, which the questions calls $\alpha$, so this probability is the length of that arc divided by the full circumference of the circle."
```

Comment:
This explanation will need to be updated now that the question component is changing. It'll be a silent error if not addressed (which is actually a separate issue: #99).